leadership.go

```diff
@@ -48,7 +50,7 @@ func NewLock(bucket *storage.BucketHandle, id, path string, ttl time.Duration, l
 		logger:                   logContext,
 		mutex:                    sync.Mutex{},
 		refreshMetadata:          false,
-		latestMetadataGeneration: 0,
+		latestMetadataGeneration: 1,
```

Comment:
Why does this need to be changed, given that it'll get overwritten when `createLock` is called?